backend/file_management/file_management_helper.py

```diff
@@ -185,24 +185,32 @@ def fetch_file_contents(file_system: UnstractFileSystem, file_path: str) -> Any:
     def delete_file(file_system: UnstractFileSystem, path: str, file_name: str) -> bool:
         fs = file_system.get_fsspec_fs()
 
-        file_path = f"{path}"
-        try:
-            if file_system.path and (not path or path == "/"):
-                file_path = f"{file_system.path}/"
-        except AttributeError:
-            if fs.path and (not path or path == "/"):
-                file_path = f"{fs.path}/"
+        if hasattr(file_system, "path") and (not path or path == "/"):
+            base_path = file_system.path
+        elif hasattr(fs, "path") and (not path or path == "/"):
+            base_path = fs.path
+        else:
+            base_path = path
+        base_path = base_path if base_path.endswith("/") else base_path + "/"
 
-        file_path = file_path + "/" if not file_path.endswith("/") else file_path
+        # Directories to delete the file from
+        directories = ["", "extract/", "summarize/"]
 
-        # adding filename with path
-        file_path += file_name
-        try:
-            with fs.open(file_path, mode="wb") as remote_file:
-                return remote_file.fs.delete(remote_file.path)  # type:ignore
-        except Exception as e:
-            FileManagerHelper.logger.info(f"Unable to delete file {e}")
-            raise FileDeletionFailed(f"Unable to delete file {e}")
+        base_file_name, _ = os.path.splitext(file_name)
```

Comment:
can we use this here
```
base_name = os.path.basename(file_name)
```

Reply:
@muhammad-ali-e I don't think this works in this case. I am only look for the name of the file without the extension. The suggestion you made will only return file name with extension.